2.7/s2i/bin/run

```diff
@@ -15,6 +15,13 @@ function should_migrate() {
 
 set -e
 
+APP_FILE="${APP_FILE:-app.py}"
+if [[ -f "$APP_FILE" ]]; then
+  echo "---> Running application from Python script ($APP_FILE) ..."
+  exec python "$APP_FILE"
+fi
+echo "WARNING: file '$APP_FILE' not found."
```

Comment:
I think we don't need this warning in every run of this script... It might make users think that app.py is required.